### DISCUSSION TITLE
Is there a learning path/proper tutorial and/or progressive developer guide available that starts from scratch?

### DISCUSSION THREAD

**Discussion by behrangsa** [vercel/community#739] [Help] [answered]
I couldn't find anything proper on the website.

First, cloning project templates is not working (already reported).

Then, at https://vercel.com/docs/get-started, I only see an option for deploying an existing project. Any paths for beginning from scratch?

**Comment by swarnava** [comment, marked as answer]
Hi there,

due to wide range of framework available, we don't provide a specific guideline in general to work on web development. You would generally pick a framework you would like to learn and go through their documentation such as: https://nextjs.org/learn/foundations/about-nextjs.

We will review your other issue and get back to you.

**Reply by neoword** [reply]
@swarnava -- unrelated somewhat... where can I find the actual code for the tutorial you reference above? Is it private? (not the examples, but the actual _source_ for [https://nextjs.org/learn](https://nextjs.org/learn))?
